.github/workflows/rust-standalone.yml

```diff
@@ -0,0 +1,49 @@
+name: rust
```

Comment:
Also, I think this should be merged into the original rust workflow. So it will not have multiple workflow with the same bindinges.